ace-window-posframe.el

```diff
@@ -0,0 +1,48 @@
+(defvar aw--posframe-frames '())
+
+(defvar aw-posframe-position-handler #'posframe-poshandler-window-center)
+
+(defun aw--lead-overlay-posframe (path leaf)
+  (let* ((wnd (cdr leaf))
+         (str (format "%s" (apply #'string path)))
+         ;; It's important that buffer names are not unique across
+         ;; multiple invocations: posframe becomes very slow when
+         ;; creating new frames, and so being able to reuse old ones
+         ;; makes a huge difference. What defines "able to reuse" is
+         ;; something like: a frame exists which hasn't been deleted
+         ;; (with posframe-delete) and has the same configuration as
+         ;; the requested new frame.
+         (bufname (format "*aw-posframe-buffer-%s*" path)))
+    (with-selected-window wnd
+      (push bufname aw--posframe-frames)
+      (posframe-show bufname
+                     :string str
+                     :poshandler aw-posframe-position-handler
+                     :font (face-font 'aw-leading-char-face)
+                     :foreground-color (face-foreground 'aw-leading-char-face)
+                     :background-color (face-background 'aw-leading-char-face)))))
```

Comment:
I'm excited to have this functionality in `ace-window` mainly for use with `exwm`. For that, though, I need to add the `(parent-frame . nil)` parameter to posframes. So a `defcustom aw-posframe-parameters` would be lovely.

Reply:
does (parent-frame . nil) work well?